packages/experiments/src/components/Chiclet/BaseChiclet.tsx

```diff
@@ -0,0 +1,76 @@
+import * as React from 'react';
+import {
+  BaseComponent,
+  css
+} from '../../Utilities';
+import { ChicletCard } from './ChicletCard';
+import { Chiclet } from './Chiclet';
+import { IBaseChicletProps, IChicletCardProps } from './Chiclet.types';
+import { IconButton, IButtonProps } from 'office-ui-fabric-react/lib/Button';
+
+export class BaseChiclet extends BaseComponent<IBaseChicletProps, any> {
+  public render() {
+    const { url, size, actions } = this.props;
+
+    let chicletCardProps = this.extractMetaTags(url);
+
+    return (
+      <Chiclet chicletCardProps={ chicletCardProps } size={ size ? size : 2 } actions={ actions } />
+    );
+  }
+
+  public extractMetaTags(url: string) {
```

Comment:
I'd put this one and the next method in a separate OpenGraph.ts file as static methods.